src/content/guides/how-to-register-an-ethereum-account/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: How to "register" an Ethereum account
+title: How to create an Ethereum account
```

Comment:
```suggestion
title: How to "create" an Ethereum account
```

Just updating based on discussion

Reply:
@corwintines Looks like this was pulled in, but I might still push back on quoting this... the only reason we quoted "register" was because it was not technically accurate... I think "How to create an Ethereum account" is technically accurate enough to not warrant any quoting, and could add confusion by leaving it in.